src/views/Swap/SendInput.vue

```diff
@@ -69,7 +69,8 @@
           <v-popover offset="1" trigger="hover focus" class="mr-2">
             <button
               :class="{
-                active: amountOption === 'min' && !disabled
+                active: amountOption === 'min' && !disabled,
+                isNotVisible: min <= 0
```

Comment:
maybe it's easier to do it with _conditional rendering_ but if when "Min" button is hidden, position of Max is changed then this is fine as well

Reply:
Yes, position of max is changed, if min is hidden